floors/validate.go

```diff
@@ -55,7 +55,7 @@ func validateFloorRulesAndLowerValidRuleKey(schema openrtb_ext.PriceFloorSchema,
 
 // validateFloorParams validates SchemaVersion, SkipRate and FloorMin
 func validateFloorParams(extFloorRules *openrtb_ext.PriceFloorRules) error {
-	if extFloorRules.Data != nil && len(extFloorRules.Data.FloorsSchemaVersion) > 0 && extFloorRules.Data.FloorsSchemaVersion != "2" {
+	if extFloorRules.Data != nil && extFloorRules.Data.FloorsSchemaVersion > 0 && extFloorRules.Data.FloorsSchemaVersion != 2 {
```

Comment:
After reading the [price floor schema 2 documentation](https://docs.prebid.org/dev-docs/modules/floors.html#schema-2) that you referenced in the description above, I have two questions:

1. According to the docs, the default value of `data.floorsSchemaVersion` is `1` and "if no schema version is provided, the module will assume version 1". Is PBS intended to not support version 1? Or why are we erroring out in line 59 when condition `extFloorRules.Data.FloorsSchemaVersion != 2` evaluates to `true`?
   Extract from the docs:
   | Param                    | Type      | Description | Default |
   |--------------------------|-----------|-------------|---------|
   | data.floorsSchemaVersion | integer   | The module supports two version of the data schema. Version 1 allows for only one model to be applied in a given data set, whereas Version 2 allows you to sample multiple models selected by supplied weights. If no schema version is provided, the module will assume version 1 for the sake of backwards compatiblity. | 1 |
1. If we indeed not support version 1, does `FloorsSchemaVersion` value of `0` have a special meaning? If not, can we simplify the condition? 
```
56   // validateFloorParams validates SchemaVersion, SkipRate and FloorMin
57   func validateFloorParams(extFloorRules *openrtb_ext.PriceFloorRules) error {
58 -     if extFloorRules.Data != nil && extFloorRules.Data.FloorsSchemaVersion > 0 && extFloorRules.Data.FloorsSchemaVersion != 2 {
   +     if extFloorRules.Data != nil && extFloorRules.Data.FloorsSchemaVersion != 2 {
59           return fmt.Errorf("Invalid FloorsSchemaVersion = '%v', supported version 2", extFloorRules.Data.FloorsSchemaVersion)
60       }
61  
floors/validate.go
```

Reply:
1. It seems to be https://docs.prebid.org/dev-docs/modules/floors.html#schema-2 needs to be updated for default value as 2, as its Schema Version 2 

2. As per Floors PRD [Floors Feature PRD](https://docs.google.com/document/d/1YAt00RfQaU2Ugciqw1LnirLgjyUz9ibfyjfxWUHR_P0/edit#heading=h.s429lun4xvfm),  Requirements for validation is 

`
Schema version validation. The feature should validate that if defined, floorsSchemaVersion is 2. This field may be missing, and if so, it's assumed that it will conform to schema 2. Unknown fields may be ignored.
`